### PR TITLE
added local dev env updated readme

### PR DESCRIPTION
**Pull request by jensendw**

### CODE REVIEW

**Comment by hewhomustnotbenamed** [comment]
I get permissions denied for getting secrets.
Here is the output from `devrun`
```
restarting the gollumtestapp so we can get a new mesos task id
{"version":"2018-06-13T12:04:06.117Z","deploymentId":"0267f970-cee2-4bf9-933c-17b843513ebf"}
waiting 15 seconds for task to restart
INFO[0000] Consul settings activated
DEBU[0000] export testkey1=testvalue1 written to /tmp/gollumtestapp
DEBU[0000] export testkey2=testvalue2 written to /tmp/gollumtestapp
DEBU[0000] export testkey3=testvalue3 written to /tmp/gollumtestapp
INFO[0000] Wrote 3 consul entries to /tmp/gollumtestapp
INFO[0000] Vault settings activated
DEBU[0000] Received unwrapped token from vault: 7704405e-72e1-72c8-74d9-28e50c4296fd
INFO[0000] Succesfully unwrapped token from vault
ERRO[0000] Problem getting secrets from vault: Error making API request.

URL: GET http://localhost:8200/v1/secret/gollumtestapp/secrets
Code: 403. Errors:

* permission denied
Contents of /tmp/gollumtestapp
export testkey1=testvalue1
export testkey2=testvalue2
export testkey3=testvalue3
```

File: devsetup.sh [hunk 1]
```diff
@@ -28,6 +28,9 @@ if [[ $CREATE ]]; then
   echo "running docker-compose"
   docker-compose up -d
 
+  echo "waiting 60 seconds for docker-compose stack to start"
```
Comment (tabossert) [inline devsetup.sh]: Please add sleep emoji here, make it easier to read

**Comment by jensendw** [comment]
@hewhomustnotbenamed do you have your output from the devsetup.sh script?  I think something may have gone wrong when provisioning the environment.  Can you do a 

```
./devsetup.sh down 
./devsetup.sh up 
````
and provide the output you're getting.  the 403 seems to be because you're not getting the correct policy.  If you want to debug further do a 
```
vault token lookup <token> 
```
This will show what policy is attached to the token you're getting back from response wrapping

File: devsetup.sh (hunk 1 above)
Comment (jensendw) [reply devsetup.sh]: @tabossert perhaps we should make https://github.com/cmstrickland/emo a requirement!

Comment (tabossert) [reply devsetup.sh]: 🥇

**Pull request merged**